src/win/path_util.cc

```diff
@@ -2,21 +2,22 @@
  * Copyright (c) 2013-2015, Christopher Jeffrey, Peter Sunde (MIT License)
  * Copyright (c) 2016, Daniel Imms (MIT License).
  * Copyright (c) 2018, Microsoft Corporation (MIT License).
+ *
+ * Ported to N-API by Matthew Denninghoff and David Russo
+ * Reference: https://github.com/nodejs/node-addon-api
+ *
```

Comment:
We can add this to the big files if you want since this was a lot of effort, but it's a little overkill for the small ones. You'll be in commit history 🙂

Reply:
Sounds good!